src/stories/gallery.stories.js

```diff
@@ -33,23 +33,42 @@ export const Fullscreen = ( { lang, dir } ) => {
 		{
 			caption: 'Wikipedia logo',
 			thumb: 'https://upload.wikimedia.org/wikipedia/commons/thumb/8/80/Wikipedia-logo-v2.svg/225px-Wikipedia-logo-v2.svg.png',
-			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/8/80/Wikipedia-logo-v2.svg/225px-Wikipedia-logo-v2.svg.png'
+			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/8/80/Wikipedia-logo-v2.svg/225px-Wikipedia-logo-v2.svg.png',
+			title: 'File:Wikipedia-logo-v2.png'
 		},
 		{
 			caption: 'Wikipedia Homepage',
 			thumb: 'https://upload.wikimedia.org/wikipedia/commons/thumb/8/87/Wikipedia_Main_Page.png/960px-Wikipedia_Main_Page.png',
-			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/8/87/Wikipedia_Main_Page.png/960px-Wikipedia_Main_Page.png'
+			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/8/87/Wikipedia_Main_Page.png/960px-Wikipedia_Main_Page.png',
+			title: 'File:Wikipedia_Main_Page.png'
 		},
 		{
 			caption: 'Wikipedia originally developed from another encyclopedia project called Nupedia',
 			thumb: 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Nupedia_logo_and_wordmark.png/480px-Nupedia_logo_and_wordmark.png',
-			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Nupedia_logo_and_wordmark.png/480px-Nupedia_logo_and_wordmark.png'
+			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Nupedia_logo_and_wordmark.png/480px-Nupedia_logo_and_wordmark.png',
+			title: 'File:Nupedia_logo_and_wordmark.png'
 		},
 		{
 			caption: 'The Wikipedia Page on December 17, 2001',
 			thumb: 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c4/English_Wikipedia_main_page_20011217.jpg/480px-English_Wikipedia_main_page_20011217.jpg',
-			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c4/English_Wikipedia_main_page_20011217.jpg/480px-English_Wikipedia_main_page_20011217.jpg'
+			src: 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c4/English_Wikipedia_main_page_20011217.jpg/480px-English_Wikipedia_main_page_20011217.jpg',
+			title: 'File:English_Wikipedia_main_page_20011217.jpg'
+		}
+	]
+
+	showFullscreenGallery( mediaItems, '', lang, dir, container )
+
+	return container
+}
 
+export const Error = ( { lang, dir } ) => {
```

Comment:
This renders as just black for me.

Reply:
Same for me

Reply:
I remember sometimes I also see a blank error page, but I can't reproduce it now.

can you load the error state directly in http://localhost:6006/?path=/story/gallery--error? checkout the fullscreen story first then error.

Reply:
well, please try it again with the latest update

Reply:
It works for me now but it's weird. It should work with any number of images.

Reply:
yes, this is fixed here! https://github.com/wikimedia/wikipedia-preview/pull/93